funktionale-try/src/test/kotlin/org/funktionale/tries/TryTest.kt

```diff
@@ -54,9 +56,62 @@ class TryTest {
         assertEquals(failure.orElse { Success(5) }.get(), 5)
     }
 
-    @Test fun foreach() {
-        success.foreach { assertEquals(10, it) }
-        failure.foreach { fail() }
+    @Test fun foreach_success_sideEffect() {
```

Comment:
I was trying to not differ toooo much from original naming. OK, I can change it.